### PR TITLE
dumpsys: add Chinese translation

### PR DESCRIPTION
**Pull request by BillLucky**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.zh/android/dumpsys.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- 列出服务的指定服务参数：
+
+`dumpsys {{service}} -h`
+
+- 从诊断输出中排除指定服务：
+
+`dumpsys --skip {{service}}`
+
+- 指定超时时间，以秒为单位（默认为10s）：
```
Comment (bl-ue) [inline pages.zh/android/dumpsys.md:27]: Missing space?
```suggestion
- 指定超时时间，以秒为单位（默认为 10s）：
```

File: pages.zh/android/dumpsys.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- 获取所有系统服务的诊断输出：
+
+`dumpsys`
+
+- 获取指定系统服务的诊断输出：
+
+`dumpsys {{service}}`
+
+- 列出 `dumpsys`可以提供的所有服务信息：
```
Comment (bl-ue) [inline pages.zh/android/dumpsys.md:15]:
```suggestion
- 列出 `dumpsys` 可以提供的所有服务信息：
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Review by bl-ue — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]

File: pages.zh/android/dumpsys.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+> 此命令只能通过 `adb shell` 使用。
+> 更多信息见：https://developer.android.com/studio/command-line/dumpsys>.
+
+- 获取所有系统服务的诊断输出：
+
+`dumpsys`
+
+- 获取指定系统服务的诊断输出：
+
+`dumpsys {{service}}`
```
Comment (navarroaxel) [inline pages.zh/android/dumpsys.md:13]: You can translate the argument option to help non-English speakers.

Comment (marchersimon) [reply pages.zh/android/dumpsys.md:13]: Should we maybe make a general rule if placeholders for command arguments should be translated too?

Comment (navarroaxel) [reply pages.zh/android/dumpsys.md:13]: Yeah, why not? If it's a placeholder it should be translated.

Comment (BillLucky) [reply pages.zh/android/dumpsys.md:13]: like this: https://github.com/tldr-pages/tldr/pull/6019#discussion_r637554415

And actually the placeholder service has been translated. It is a '服务'. Do we need to highlight it or use backticks to distinguish it?

Thanks for review.  @navarroaxel @marchersimon

Comment (marchersimon) [reply pages.zh/android/dumpsys.md:13]: I was thinking of this:
```suggestion
`dumpsys {{服务}}`
```

**Review by BillLucky — COMMENTED** [review]
`service` has been translated @navarroaxel

File: pages.zh/android/dumpsys.md (hunk 3 above)
Comment (bl-ue) [reply pages.zh/android/dumpsys.md:13]: @BillLucky yes, what @marchersimon suggested is what @navarroaxel is getting at, and what I would think would be the best thing to do.  **But**, you raise an interesting point.  Do most Chinese people have to translate it into English in their minds in order to use it, like you're saying?

Comment (navarroaxel) [reply pages.zh/android/dumpsys.md:13]: @BillLucky yeah, we do that for translations.

**Comment by navarroaxel** [comment]
> `service` has been translated @navarroaxel

I don't see new commit/changes :thinking:

**Comment by navarroaxel** [comment]
> `service` has been translated @navarroaxel

@BillLucky I don't see new commit/changes :thinking:

**Comment by BillLucky** [comment]
> > `service` has been translated @navarroaxel
> 
> @BillLucky I don't see new commit/changes 🤔

Oh, because `service` is Chinese `服务`  .
the previous translation has already included @navarroaxel

**Review by navarroaxel — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Pull request merged**